css/style.css

```diff
@@ -68,12 +121,87 @@ body {
 	padding-right: 5px;
 }
 
+/* Image slideshow CSS begins */
 .slider {
 	width: 100%;
 	height: 300px;
-	background: #000;
```

Comment:
I removed the background colour when adding the javascript for the image slider because I will not be needing a specified background colour when the images get put in